docs/katex-math-rendering.md

```diff
@@ -0,0 +1,58 @@
+# Katex Math Rendering
+
+Apart from using the method mentioned in [[math-support]], we can also use KaTeX to render our math equations in Foam. The caveat is: we can't rely on GitHub Pages to host and deploy our website anymore, because the plugin we'll be using to let Jekyll support KaTeX doesn't play well together with GitHub Pages.
```

Comment:
Is it actually the case that GitHub pages doesn't support KaTeX? Or is the issue simply that the default Primer template does not include the katex Jekyll plugin, but if it were added, it would work in the GitHub Pages deployment pipeline?

I would imagine you could do the same "ejection" by including the Jekyll config files and still publish to GitHub pages. I haven't however tested this myself?

Reply:
@jevakallio This is indeed a GitHub Pages issue as stated in this issue: [jekyll-katex/issues/33](https://github.com/linjer/jekyll-katex/issues/33). I myself encountered exactly the same error messages when trying to deploy the same repository to GitHub Pages so I would assume it's a GitHub Pages issue.

Reply:
Oh, interesting! I don't have the time to see if I could get this to work right now, so I'll take your word for it.